src/mmw/js/src/analyze/views.js

```diff
@@ -237,7 +237,7 @@ var TabContentView = Marionette.LayoutView.extend({
 
     showResults: function() {
         var name = this.model.get('name'),
-            results = JSON.parse(this.model.get('taskRunner').get('result')),
+            results = JSON.parse(this.model.get('taskRunner').get('result')).survey,
```

Comment:
While the `analyze` endpoint now returns both census and survey, the other endpoints `analyze/animals` and `analyze/pointsources` don't, causing those tables in the Analyze tab to be empty even when there is real data.